openbb_terminal/economy/economy_controller.py

```diff
@@ -1000,6 +1023,98 @@ def call_ycrv(self, other_args: List[str]):
                     export=ns_parser.export,
                 )
 
+    @log_start_end(log=logger)
+    def call_ecocal(self, other_args: List[str]):
+        """Process ecocal command"""
+        parser = argparse.ArgumentParser(
+            add_help=False,
+            formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+            prog="ecocal",
+            description="Economic calendar.",
+        )
+        parser.add_argument(
+            "-tz",
+            "--timezone",
+            action="store",
+            dest="time_zone",
+            nargs="+",
+            default=None,
+            help="Time zone in GMT +/-h:mm format. E.g. GMT +1:00",
+        )
+        parser.add_argument(
+            "-c",
+            "--country",
+            action="store",
+            dest="country",
+            nargs="+",
+            default="united states",
+            help="Display calendar for specific country.",
+        )
+        parser.add_argument(
+            "-i",
+            "--importances",
+            action="store",
+            dest="importances",
+            default=None,
+            help="Event importance classified as high, medium or low.",
+        )
+        parser.add_argument(
+            "-cat",
+            "--categories",
+            action="store",
+            dest="categories",
+            nargs="+",
+            default=None,
+            help="Event category.",
```

Comment:
you can add the param `choices` to restrict only certain values, and to show in documents the available options. Like the `interval` one below. You already added auto-completion for this purpose, but argparse wouldn't allow for autocompletion if it's not the first argument. Pain in the neck.

<img width="413" alt="image" src="https://user-images.githubusercontent.com/40023817/177218072-8d8d7cee-b149-4b8c-a974-7d06226a06a2.png">